rj_gameplay/stp/play/__init__.py

```diff
@@ -87,41 +90,45 @@ def assign_roles(
 
         used_robots = set()
         for tactic in self.prioritized_tactics:
-            # TODO: handle if tactic requests more roles than exist
-            #       by not assigning tactic at all!
-            #       and give some default behavior (later)
             robots_for_tactic = []
```

Comment:
Could you leave a comment explaining what robots_for_tactic represents/how the behavior has changed? I believe currently that it's something like "check if all the robots a tactic requests can be given to it: if yes, assign them all, if no, don't assign any of them"